aspnetcore/blazor/debug.md

```diff
@@ -432,6 +432,10 @@ VsRegEdit.exe set "<VSInstallFolder>" HKCU JSDebugger\Options\Debugging "BlazorT
 
 The `{TIMEOUT}` placeholder in the preceding command is in milliseconds. For example, one minute is assigned as `60000`.
 
+### Protocol exception
+
+If you receive an exception of type `Microsoft.VisualStudio.Shared.VSCodeDebugProtocol.ProtocolException`, avoid using [Fiddler](https://www.telerik.com/fiddler) while debugging.
```

Comment:
So, VS will throw the `ProtocolException` for a variety of issues related to not being able to connect to the debug server.

It's not always related to Fiddler, and I haven't debugged that this is for sure something that is supported so not sure about including this in the docs.